### PR TITLE
Add background to the text.

### LINKED ISSUE #2: Awesome extension! The White Text is hard to see with the new 2019 logo.
The White Text is hard to see with the new 2019 logo. Wondering if black would be better, but then hard to see behind the logo. Not sure the best answer.

### PR DESCRIPTION
**Pull request by yauhenipakala**
Hi @alexsorokoletov, thank you for the awesome addin!

Propose you some changes:
- Upgrade project to the latest VS4Mac 8+
- Add build script for packaging addin
- Add background to text. Fixes #2
- Update version
- Add new release package for VS4Mac 8.4.5+

Preview:
Screen Shot 2020-02-16 at 15 17 43

Also, I created a test project for fast debug the solution (screenshot above), I'll can to add to the repository in the next PR. What do you think about it?

macOS Dock:
dock_bottom
dock_left
macOS app switcher:
switcher

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,3 +39,5 @@ Thumbs.db
 # dotCover
 *.dotCover
 .droidres/
+
+.vs/
```
Comment (alexsorokoletov) [inline .gitignore:43]: 👍

File: SolutionName/SolutionName.csproj [hunk 2]
```diff
@@ -49,11 +33,7 @@
 		<EmbeddedResource Include="Properties\Manifest.addin.xml" />
 	</ItemGroup>
 	<ItemGroup>
-		<AddinReference Include="MonoDevelop.MacPlatform" />
-	</ItemGroup>
-	<ItemGroup>
-		<None Include="packages.config" />
+		<PackageReference Include="MonoDevelop.Addins" Version="0.4.7" />
```
Comment (alexsorokoletov) [inline SolutionName/SolutionName.csproj:36]: 👍 one file less

File: pack.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+VSTOOL_PATH="/Applications/Visual Studio.app/Contents/Resources/lib/monodevelop/bin/vstool.exe"
```
Comment (alexsorokoletov) [inline pack.sh:3]: That's great, I'd love to see how we can actually set up a full CI that would rebuild the extension with any new VS4Mac version.

**Review by alexsorokoletov — APPROVED** [review]
PR is good to go. Here are some questions/nice-to-haves:
- how does a multiline string looks with the background? Preview (thanks for that!) shows only one line.
- it would be great to have a preview of how that change looks in MacOS switcher (it kinda has a different bg itself than the Preview window)
- Can you include the preview app you created for the preview?
- Would you be kind as to update the `README.md` showing the new style and the `Other notes` section with how to build/pack the extension now that you have a script?
https://github.com/DreamTeamMobile/VS4Mac.SolutionName#other-notes

Awesome contribution and very much appreciated!

**Comment by alexsorokoletov** [comment]
@wcoder Hey!
I think this is a great initiative from you! :)

Let me know if I should merge this in or maybe you have some extras coming in (see my comments).
Thank you!

**Comment by yauhenipakala** [comment]
Hi @alexsorokoletov thanks for the comments!

> how does a multiline string looks with the background? Preview (thanks for that!) shows only one line.

Done (to PR description and Meta folder)

> it would be great to have a preview of how that change looks in MacOS switcher (it kinda has a different bg itself than the Preview window)

Done (to PR description and Meta folder)

> Can you include the preview app you created for the preview?

I'd add in a separate PR.

> Would you be kind as to update the README.md showing the new style and the Other notes section with how to build/pack the extension now that you have a script?

Done.

**Pull request merged**